src/components/UserInfo.js

```diff
@@ -1,20 +1,31 @@
 export default class UserInfo {
-  constructor({ title, description }) {
+  constructor({ title, description, link }) {
     this._title = document.querySelector(title);
     this._description = document.querySelector(description);
-    this._closeBtn = document.querySelector("#edit-close-button");
+    this._avatar = document.querySelector(link);
+    this._profileImageButton = document.querySelector(".profile__image-button");
```

Comment:
`    this._profileImageButton = document.querySelector(".profile__image-button");
`

Please, delete it because you don't use it